src/client/ActionIndicatorCell.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2024 Inclusive Design Research Centre, OCAD University
```

Comment:
The years in copyright should be updated to 2024-2025 now. :)

Reply:
Slavery!  :-)

Should I do this for all of the files, or just the ones that are part of this PR.  That is, only the ones that have been modified?

Reply:
It might be easier to do a find and replace across all files. If you think only files touched in this PR should be updated, I'm fine too.

Reply:
I manually changed some of the copyright dates of the files in this PR as I made code changes to that file. Then I manually fixed the rest (10 files) in one go, but only those in this PR.  I'm not sure what the pattern is to look for in order to automate this.  I left the rest alone.